homeassistant/components/switch/flux.py

```diff
@@ -111,7 +112,8 @@ def update(call=None):
         """Update lights."""
         flux.flux_update()
 
-    hass.services.register(DOMAIN, name + '_update', update)
+    service_name = slugify("{} {}".format(name, 'update'))
+    hass.services.register(DOMAIN, service_name, update)
```

Comment:
Why we make a server per flux "device"? We can store the device on `hass.data` and register the servcie on first call. After that we can use the servcie like others with entity_ids. Look to sonos, how that can work.